packages/wrangler/src/dev/use-esbuild.ts

```diff
@@ -172,6 +174,8 @@ export function runBuild(
 						// sourcemap defaults to true in dev
 						sourcemap: undefined,
 						checkFetch,
+
+						metafile,
```

Comment:
```suggestion
						metafile: undefined,
```

this function is used in wrangler dev, which we haven't added this flag to, so let's just set it to undefined here.